### ISSUE TITLE
Snackbar: the `aria-atomic` tags don't seem to be correct

### ISSUE THREAD

**Issue by Kiarokh** [Lundalogik/lime-elements#2014]
## Current behavior
the `aria-atomic` is now on two places, one is always false and one is always true
![image](https://user-images.githubusercontent.com/35954987/211299765-fdbad4e0-997b-46cb-9f7c-aeacfe32ce3f.png)

## Expected behavior
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-atomic
I think the `aria-atomic` tag should be set on the entire component, and it should turn `true` as soon as the Snackbar is displayed.
